src/constants.lua

```diff
@@ -1,17 +1,23 @@
 local constants = {}
 
--- GAR
-constants.DEFAULT_UNDERNAME_COUNT = 10
-constants.DEADLINE_DURATION_MS = 60 * 60 * 1000 -- One hour of miliseconds
 constants.oneYearSeconds = 60 * 60 * 24 * 365
 constants.thirtyDaysSeconds = 60 * 60 * 24 * 30
-constants.defaultundernameLimit = 10
+
+constants.oneHourMs = 1000 * 60 * 60
+constants.oneDayMs = constants.oneHourMs * 24
+constants.oneWeekMs = constants.oneDayMs * 7
+constants.twoWeeksMs = constants.oneWeekMs * 2
+constants.oneYearMs = 31536000 * 1000
+
+-- GAR
+constants.DEFAULT_UNDERNAME_COUNT = 10
+constants.DEADLINE_DURATION_MS = constants.oneHourMs
 constants.totalTokenSupply = 1000000000 * 1000000 -- 1 billion tokens
 constants.MIN_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.10 -- the minimum penalty rate for an expedited withdrawal (10% of the amount being withdrawn)
 constants.MAX_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.50 -- the maximum penalty rate for an expedited withdrawal (50% of the amount being withdrawn)
 constants.mIOPerIO = 1000000
 constants.minimumWithdrawalAmount = constants.mIOPerIO -- the minimum amount that can be withdrawn from the GAR
-constants.redelegationFeeResetIntervalMs = 1000 * 60 * 60 * 24 * 7 -- 7 epochs
+constants.redelegationFeeResetIntervalMs = constants.oneWeekMs -- 7 epochs
```

Comment:
Consider defining other constants to enable:
```
constants.redelegationFeeResetIntervalMs = 7 * constants.msPerEpoch
```

Reply:
Nit that "DurationMs" would be better in the name than "Length".

Reply:
c5c7c0e